docs-conceptual/azps-13.2.0/mfa-requirement.md

```diff
@@ -0,0 +1,88 @@
+---
+description: Learn about the impact of mandatory multifactor authentication enforcement on Azure PowerShell in automation scenarios
+ms.custom: devx-track-azurepowershell
+ms.devlang: powershell
+ms.service: azure-powershell
+ms.topic: conceptual
+title: The impact of multifactor authentication on Azure PowerShell in automation scenarios
+---
+
+# The impact of multifactor authentication on Azure PowerShell in automation scenarios
+
+This article explores how multifactor authentication (MFA) affects automation tasks that use
+Microsoft Entra user identities and provides guidance on alternative approaches for uninterrupted
+automation.
+
+> [!IMPORTANT]
+> Action is required if you're using Microsoft Entra user identities for automation.
+
+MFA requirements prevent you from using Microsoft Entra user identities for authentication in
+automation scenarios. Organizations must switch to authentication methods designed for automation,
+such as managed identities or service principals, which support non-interactive automation use
+cases.
+
+## Limitations of user identities with MFA in automation
+
+- **Interactive authentication**: MFA is triggered during interactive sign-ins when using a
+  Microsoft Entra user identity. For automation scripts relying on a user identity, MFA disrupts the
+  process because it requires extra verification steps. For example, authenticator app, phone call,
+  etc. that you can't automate. This verification prevents automation from running unless
+  authentication is handled in a non-interactive way, such as with a managed identity or service
+  principal.
+
+- **Scripted login failures**: In automation scenarios like running Azure PowerShell scripts
+  unattended, an MFA-enabled user identity causes the script to fail when trying to authenticate.
+  Since MFA requires user interaction, it's incompatible with non-interactive scripts. This means
+  you must switch to a managed identity or service principal, both of which use non-interactive
+  authentication.
+
+- **Security considerations**: While MFA adds an extra layer of security, it can limit automation
+  flexibility, especially in production environments where automation must run without manual
+  intervention. Shifting to managed identities or service principals, which don't require MFA, is
+  more practical and secure in such environments.
```

Comment:
Should we indicate here that SP/ Managed Identity / Federated Identity have been made for automation purposes?

Reply:
I added the mention of federated identities and that all three are designed for automation purposes.